### DISCUSSION TITLE
help:  I lost 20 Chia

### DISCUSSION THREAD

**Discussion by nodo33** [Chia-Network/chia-blockchain#15924] [Support]
Hello, if you can help me, I have a chia farm from which 20 pieces of chia have disappeared. Similar transactions do not appear in the transfers. The problem appeared when I switched to the new 1.8.2 version.

**Comment by BrandtH22** [comment]
Good afternoon @nodo33 , 

I hope this message finds you well.As a heads up the majority of our support is currently provided on our discord server (https://discord.gg/chia) in the #support channel.
There are two main reasons why xch that was in your wallet would no longer be there:

1. The keyset or device has been compromised, in this case we at Chia Network unfortunately cannot assist other than recommending that security best practices be followed to make yourself a hard target. More information on how to do this can be found in this article: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

2. The client failed to sync or has a corrupt database. In this case we recommend first trying the GUI based resync under settings/ advanced. If the coins are still not visible then we would recommend forcing a full resync by:
- Closing the Chia client
- Renaming the wallet database (rename to archive as this will remove all pending offers and CAT/DID names from the wallet)
 - note that the wallet db is under the directory .chia/mainnet/wallet/db/ and is listed with the wallet fingerprint as an identifier
- Restarting the Chia client and waiting for sync
